README.md

```diff
@@ -193,3 +193,14 @@ type: kubernetes.io/dockerconfigjson
 data:
   .dockerconfigjson: e30K
 ```
+
+#### Special case: Resource with .metadata.ownerReferences
+
+Sometimes, secrets are generated by external components. Such secrets are configured with an ownerReference. By default, the kubernetes-replicator will copy the 
+ownerReference to the target namespace, too.
+
+ownerReference won't work [across different namespaces](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents) and the secret at the destination will be removed by the kubernetes garbage collection.
+
+By set `-strip-owner-reference=true` as CLI argument, kubernetes-replicator will strip the ownerReference fields to avoid deletion of the secrets in the destination.
```

Comment:
Maybe we should control this decision via annotations and thus have the possibility to decide per secret if the OwnerRef should be copied or not. Whereby the default behavior should be that the OnwerRef is not copied.

What do you think?

Reply:
Good idea, See comment below.